### PR TITLE
Buf fix - Read only fix

### PR DESCRIPTION
**Pull request by Regis18**

### CODE REVIEW

File: src/main/java/com/jala/model/search/SearchBasic.java [hunk 1]
```diff
@@ -34,111 +34,110 @@
...
-                            getFileDate(file,"Creation"),
-                            getFileDate(file,"Modification"),
-                            getFileDate(file, "LastAccess"));
-                    result.add(asset);
-                }
-            }
-        }
-        return result;
-    }
+	public List<Asset> search(CriteriaSearch criteria) {
```
Comment (AKatrin) [inline src/main/java/com/jala/model/search/SearchBasic.java:58]: Please do format code

File: src/test/java/com/jala/model/search/SearchBasicTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class SearchBasicTest {
+
+	private SearchCommon search;
+	private List<Asset> list = new ArrayList<>();
+	private CriteriaSearch criteriaSearch;
+	@Before
```
Comment (raulchoque-eng) [inline src/test/java/com/jala/model/search/SearchBasicTest.java:36]: Add an Space above this line

File: src/test/java/com/jala/model/search/SearchBasicTest.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	@Test
+	public void search_ReturnHidden_WhenSentReadTrue() {
+		criteriaSearch.setReadOnly(TernaryBooleanEnum.OnlyTrue);
+		search = new SearchCommon(criteriaSearch);
+		list = search.search();
+		for (int i = 0; i < list.size(); i++) {
+			System.out.println("FIN");
+			System.out.println(list.get(i).getPath());
+			System.out.println(list.get(i).isReadOnly());
+
```
Comment (raulchoque-eng) [inline src/test/java/com/jala/model/search/SearchBasicTest.java]: Clear this line

**Review by raulchoque-eng — APPROVED** [review]

File: src/test/java/com/jala/model/search/SearchBasicTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+		criteriaSearch = new CriteriaSearch(path);
+	}
+
+	@Test
+	public void search() {
+		search = new SearchCommon(criteriaSearch);
+		list = search.search();
+		for (int i = 0; i < list.size(); i++) {
+			System.out.println(list.get(i).getPath());
+			System.out.println(list.get(i).isReadOnly());
```
Comment (AKatrin) [inline src/test/java/com/jala/model/search/SearchBasicTest.java:48]: Add assert to the tests

**Pull request merged**